### PR TITLE
Fixed some bugs and added some features

### PR DESCRIPTION
**Pull request by kaminomisosiru**
### Bug fix

- Fix some typo.
- Changed to display some text in the language appropriate to the system language.
- Fixed an issue where moving files from the left panel to the right panel files are not displayed in the panel.
- Directory is loaded recursively so that the directory of arbitrary depth can be loaded.

### add new features

- When taking a screen shot of the analysis result, display a dialog for selecting a place to save.
- The program was terminated when the "Clear" button in the main window was pressed

### CODE REVIEW

**Comment by manuel-freire** [comment]
Wow, looks nice - great work. I will need some time to review it before merging, though.

File: ac-core/src/main/java/es/ucm/fdi/ac/test/TokenCountTest.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public void preprocess(Submission s) {
 		String tokens = getTokens(s);
 
 		TreeMap<Integer, Integer> counter = new TreeMap<Integer, Integer>();
-		StringTokenizer st = new StringTokenizer(tokens, "\n\r");
+		StringTokenizer st = new StringTokenizer(tokens, " ");
```
Comment (manuel-freire) [inline ac-core/src/main/java/es/ucm/fdi/ac/test/TokenCountTest.java]: typical antlr4 tokenized strings look like `3a 13 9 36 1r 3b 1s -1` (for the `sample.java` file included in ac-lexers test), so yes, this works. The old tokenizer (from PMD) generated output in columns. I would, however, prefer to use more whitespace characters to future-proof things somewhat: " \n\r\t"

File: ac-core/src/main/java/es/ucm/fdi/ac/Analysis.java [hunk 2]
```diff
@@ -150,6 +134,34 @@ public void loadSources(SourceSet sources) throws IOException {
 		}
 	}
 
+	private void addSubmissionRecursive(FileTreeNode tn,
+			HashMap<String, Submission> unique, int i) {
+		if (tn.getFile().isDirectory()) {
+			for (FileTreeNode fn : tn.getLeafChildren()) {
+				addSubmissionRecursive(fn, unique, i);
```
Comment (manuel-freire) [inline ac-core/src/main/java/es/ucm/fdi/ac/Analysis.java]: if `i` is incremented inside recursive calls, these increments will be invisible once the calls return. This results in duplicate ids being assigned, which is bad. Given that we have the `unique` hashmap, we can dispose with i, and just use the hashmap's current size as unique identifiers. In any case, as it is, this code is buggy.

**Comment by kaminomisosiru** [comment]
Thank you for your review.
I have fixed the improvements you pointed out.

- In addition to blanks, I added delimiters "\n\r\t".
- The number of submissions is counted by a static variable.

**Pull request closed without merging**